src/java/main/org/apache/zookeeper/server/PurgeTxnLog.java

```diff
@@ -108,9 +141,11 @@ public boolean accept(File f){
         // remove the old files
         for(File f: files)
         {
-            System.out.println("Removing file: "+
+            final String msg = "Removing file: "+
                 DateFormat.getDateTimeInstance().format(f.lastModified())+
-                "\t"+f.getPath());
+                "\t"+f.getPath();
+            LOG.info(msg);
+            System.out.println(msg);
```

Comment:
It's not ideal but they both serve a purpose that the other cannot as far as I can tell.

System.out.println is useful when this is invoked directly through CLI.  The old behavior was to provide just this.
LOG.info is useful in that it's visible in the server log.  The old behavior did not log this info which makes debugging through server logs harder.